x-pack/plugin/shutdown/src/internalClusterTest/java/org/elasticsearch/xpack/shutdown/NodeShutdownShardsIT.java

```diff
@@ -536,9 +334,9 @@ public void testAutoExpandDuringRestart() throws Exception {
             );
         });
 
-        client().prepareIndex("myindex").setSource("field", "value");
+        indexRandomData("myindex");
```

Comment:
Hmm I think the previous line of code was basically a no-op whereas with this change we're now actually indexing stuff.

Reply:
I confirmed with original author that this was intended to index